icinga2x/Vagrantfile

```diff
@@ -36,7 +37,6 @@ nodes = { 'icinga2' => {
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   nodes.each_pair do |name, options|
     config.vm.define name do |node_config|
-      node_config.vm.box = options[:box_virtualbox]
```

Comment:
Removing this default will break other providers, if there is no openstack plugin installed.